### DISCUSSION TITLE
我感觉可以提个意见

### DISCUSSION THREAD

**Discussion by ghost** [imsyy/SNav#23] [Ideas]
一,
在做一个可定制位置或固定位置的音乐播放器伸缩就像你的Home项目一样
二,
快捷方式,应该可以做一下更改,做一个按钮点击后即可直达所标记URL网址

**Comment by ghost** [comment]
另外你的Home项目是占时不发布了吗?我下载以后发现无法正常运作
还是说我的使用方法是错误的.

**Reply by DogTang0323** [reply]
可以正常运行呀

**Comment by loongworld** [comment]
大哥，还更新吗，期待(*^o^*)！
